programs/drift/src/controller/orders.rs

```diff
@@ -1665,36 +1656,22 @@ fn get_referrer<'a>(
     Ok((Some(referrer), Some(referrer_stats)))
 }
 
-fn determine_if_user_order_is_risk_decreasing(
+fn determine_if_user_order_is_position_decreasing(
     user: &User,
     market_index: u16,
     order_index: usize,
 ) -> DriftResult<bool> {
     let position_index = get_position_index(&user.perp_positions, market_index)?;
     let order_direction = user.orders[order_index].direction;
     let position_base_asset_amount_before = user.perp_positions[position_index].base_asset_amount;
-    is_order_risk_decreasing(
+    is_order_position_reducing(
```

Comment:
decided better to only check maintenance if it is position reducing, not "risk" reducing

Reply:
yea this matches reduce_only flag